setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='service-configuration-lib',
-    version='2.5.6',
+    version='2.5.7',
```

Comment:
```suggestion
    version='2.5.6',
```
we normally bump the version outside of a PR (and then tag that commit in which we bumped the verison)